roles/ceilometer-common/defaults/main.yml

```diff
@@ -38,26 +38,48 @@ ceilometer:
   heartbeat_timeout_threshold: 30
   collector_workers: 4
   logs:
-    - paths:
-        - /var/log/ceilometer/api.log
-      fields:
-        type: openstack
-        tags: ceilometer,ceilometer-api
-    - paths:
-        - /var/log/ceilometer/agent-notification.log
-      fields:
-        type: openstack
-        tags: ceilometer,ceilometer-agent-notification
-    - paths:
-        - /var/log/ceilometer/collector.log
-      fields:
-        type: openstack
-        tags: ceilometer,ceilometer-collector
-    - paths:
-        - /var/log/ceilometer/polling.log
-      fields:
-        type: openstack
-        tags: ceilometer,ceilometer-polling
+    ubuntu:
```

Comment:
what decides the file to log to ?  surely this is something we can configure in the ceilometer config file rather than having different paths for different OS ?   is this def a ubuntu vs rhel thing, or could it be changed locations from mitaka to newton ?